src/app/common/utils/spam-filter.ts

```diff
@@ -0,0 +1,23 @@
+const urlRegex =/(((http|ftp|https):\/\/)?(((http|ftp|https):\/\/)?(([\w.-]*)\.([\w]*))))/g;
+const spamWords = ["won", "win", "click"];
+export const spamReplacement = 'Unknown token';
+
+function spamUrlFilter(input: string): boolean {
+    return input.match(urlRegex) ? true : false;
+}
+
+function spamWordFilter(input: string): boolean {
+    const spam = (element: string) => input.toLowerCase().includes(element);
```

Comment:
Use an action/verb name for this fn? `containsSpam`? Then it reads really well on next line `spamWords.some(containsSpam)`

Reply:
Sure, I'll update, thanks